src/server/localisations/infra/repositories/apiGeo.repository.test.ts

```diff
@@ -1,57 +1,53 @@
 import { CacheAxiosResponse } from 'axios-cache-interceptor';
 
-import { createSuccess } from '~/server/errors/either';
+import { createFailure, createSuccess, Failure } from '~/server/errors/either';
+import { ErreurMétier } from '~/server/errors/erreurMétier.types';
 import { ApiGeoRepository } from '~/server/localisations/infra/repositories/apiGeo.repository';
-import { CachedHttpClientService } from '~/server/services/http/cachedHttpClient.service';
+import { aLogInformation, anErrorManagementService } from '~/server/services/error/errorManagement.fixture';
+import { anHttpError } from '~/server/services/http/httpError.fixture';
 import {
 	aCacheAxiosResponse,
 	aCachedHttpClientService,
 } from '~/server/services/http/publicHttpClient.service.fixture';
-import { LoggerService } from '~/server/services/logger.service';
-import { aLoggerService } from '~/server/services/logger.service.fixture';
 
-describe('ApiGeoLocalisationRepository', () => {
-	let httpClientService: CachedHttpClientService;
-	let loggerService: LoggerService;
-	let apiGeoLocalisationRepository: ApiGeoRepository;
-
-	beforeEach(() => {
-		httpClientService = aCachedHttpClientService();
-		loggerService = aLoggerService();
-
-		apiGeoLocalisationRepository = new ApiGeoRepository(httpClientService, loggerService);
-	});
+const aLogInformationApiGeo = aLogInformation({
+	apiSource: 'API Geo',
+	contexte: 'get localisation', message: '[API Geo] impossible de récupérer une ressource',
+});
```

Comment:
Je suis un peu dérangé par l'approche très générique sur le `aLogInformationApiGeo` avec les même contexte / message pour tous les cas 

Proposition : 
```suggestion
const aLogInformationApiGeo = (contexte, message) => aLogInformation({
	apiSource: 'API Geo',
	contexte,
	message,
});
```

quitte à avoir de la répétition de test en test (mais permettre de voir plus facilement le contenu qui est testé dans chaque test) 
+ remplacer `get localisation` par des `get departements`, `get communes`, etc. en fonction de qui est appelé 

Qu'en penses-tu?

Reply:
j'aime pas l'idée de mettre plusieurs messages différents sinon on risque d'avoir trop d'erreurs différentes sur Sentry